docs/run-keep-ecdsa.adoc

```diff
@@ -20,22 +20,27 @@ these capabilities consider the following criteria:
   application port running under the same IP.
 - Recommended machine types by provider:
 
+Your operating environment will ultimately dictate what machine type to go with.  This is
+particulary relevant if you're running a containerized solution where multiple applications are
+sharing VM resources.  The below types are sufficient for running at least one instance of the
+Keep ECDSA client.
+
 [%header,cols=2*]
 |===
 |Cloud Provider
 |Machine Type
 
 |Google Cloud
-|n1-standard-4
+|n1-standard-2
 
 |AWS
-|m5.xlarge
+|t3.medium
 
 |Azure
-|D4s v3
+|A2 v2
 
 |Self-hosted
-|4 vCPU / 4 GiB RAM / 1 GiB Persistent Storage
+|2 vCPU / 4 GiB RAM / 1 GiB Persistent Storage
```

Comment:
I think we can reduce memory requirements pretty safely to 2 GB. We may be able to reduce it even more but it requires more monitoring of nodes performance with higher network usage.

Reply:
Have you  run testnet nodes on 2 reliably?

Reply:
Yes, I've been also running them on 1GB pretty stable but don't want to recommend it just yet. 2 GB should be more than enough.

Reply:
Great thanks, updated.